common/lib/xmodule/xmodule/js/src/video/01_initialize.js

```diff
@@ -11,7 +11,16 @@
  */
 
 (function (requirejs, require, define) {
-
+require.config({
```

Comment:
@cahrens I agree that requirejs config need to be added to a central location. However we have added `rquirejs` path for `moment` here because it was not fetched correctly from the central location where requirejs config for `moment.min.js` is present but was sending path something like `/static/moment.js` in `video/01_initialize.js` which was resulting `404` instead of correct path  `/static/js/vender/moment.min.js` 

About `TNL-3311`, I am not sure, I would need to check it to see if this has any impact on it. I will get back to you once I get enough information on it.